contracts/cw4-voting/src/contract.rs

```diff
@@ -21,12 +21,13 @@ pub fn instantiate(
     deps: DepsMut,
     env: Env,
     info: MessageInfo,
-    msg: InstantiateMsg,
+    mut msg: InstantiateMsg,
 ) -> Result<Response, ContractError> {
     set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
     if msg.initial_members.is_empty() {
         return Err(ContractError::NoMembers {});
     }
+    msg.initial_members.dedup();
```

Comment:
IMO I think it would make more sense here to error if we found duplicates. I can't really think of a situation wherein somebody does that intentionally.

Reply:
agreed!